packages/nodes-base/credentials/MalcoreApi.credentials.ts

```diff
@@ -0,0 +1,51 @@
+import type {
+	IAuthenticateGeneric,
+	ICredentialTestRequest,
+	ICredentialType,
+	INodeProperties,
+} from 'n8n-workflow';
+
+export class MalcoreApi implements ICredentialType {
+	name = 'malcoreApi';
+
+	displayName = 'MalcoreAPI';
+
+	documentationUrl = 'malcore';
+
+	icon = { light: 'file:icons/Malcore.svg', dark: 'file:icons/Malcore.svg' } as const;
```

Comment:
```suggestion
	icon = { light: 'file:icons/Malcore.png', dark: 'file:icons/Malcore.png' } as const;
```

Reply:
@ShireenMissi thanks that was a rookie mistake I made 😄